### PR TITLE
Project features

### PR DESCRIPTION
**Pull request by wayungi**
### This PR adds 💯 
✅   HomePage component
✅  DetailPage component
✅  Styles to the components
✅  Fetch data from Soccer standings API
✅  Search feature

### CODE REVIEW

File: src/components/__tests__/__snapshots__/HomePage.test.js.snap [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+        <input
+          placeholder="Search by league name"
+          type="search"
+        />
+      </form>
+      <div
+        class="panels"
+      />
+    </div>
+  </div>,
```
Comment (Mihndim2020) [inline src/components/__tests__/__snapshots__/HomePage.test.js.snap:41]: - [ ] Good job with test coverage, but according to the project [requirements](https://github.com/microverseinc/curriculum-react-redux/blob/main/capstone/react_capstone.md),  " `Create unit tests for pure functions in your Redux code,  Create integration tests for your application using the React Testing Library` ", please ensure your project has unit tests for several components of the app and some pure functions in Redux code and has integration tests using React Testing Library.

**Review by Mihndim2020 — CHANGES_REQUESTED** [review]
### Hi @wayungi, 

### Good job so far!
### To Highlight! :clap:  :green_circle: 

:heavy_check_mark: Correct use of Gitflow. 
:heavy_check_mark: No `Linter` errors
:heavy_check_mark: Descriptive `PR` summary,
:heavy_check_mark: Great job with UI,
:heavy_check_mark: Clean/Organized code,
:heavy_check_mark: Great job with video presentation,
:heavy_check_mark: Good commit messages,
:heavy_check_mark: Descriptive Readme file.

There are some issues that you still need to work on to get this project approved, but you are almost there!
ALMOST-MINION

# Status: Requires Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

### N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear or you can get to me on slack  using this handle @mihndim.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by topeogunleye — APPROVED** [review]
Hi @wayungi, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

> ## (Highlights) Good Points: 💪 
> * Good commit messages. :heavy_check_mark:
> * No linter errors. :heavy_check_mark:
> * Followed Github flow :heavy_check_mark:
> * Professional README file. :heavy_check_mark:
> * PR has a good title and summary :heavy_check_mark:

çrpik-kurdî-impressive



Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear
Please, remember to tag @topeogunleye in your question so I can receive the notification.
 
As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

**Comment by wayungi** [comment]
Thank you @topeogunleye.

**Comment by topeogunleye** [comment]
> Thank you @topeogunleye.
@wayungi , you're welcome

**Pull request merged**